CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v3.4.26 (unreleased)
+ - Add cache configuration per endpoint in fallback cache
```

Comment:
There can't be two sections named `v3.4.26`! :)

Reply:
Dang...sorry!

Reply:
Fixed